components/Pages/Finance/CreateDraftPayRun/CreatePayRunInfo.js

```diff
@@ -0,0 +1,89 @@
+import { Controller } from 'react-hook-form';
+import React from 'react';
+import { Container, FormGroup, RadioGroup } from '../../../HackneyDS';
+import DatePick from '../../../DatePick';
+
+const CreatePayRunInfo = ({
```

Comment:
Break this up into two components
<img width="615" alt="Screenshot 2021-11-11 at 14 06 33" src="https://user-images.githubusercontent.com/89010589/141311777-c6f739d2-595b-4caf-8a3c-d117c9261e4c.png">
and
<img width="629" alt="Screenshot 2021-11-11 at 14 06 58" src="https://user-images.githubusercontent.com/89010589/141311809-75c09681-fdc1-4535-80a1-6fed74fb88be.png">
It is better to have two simpler components than one complicated one with lots of props. I'd expect this component to look something like

```
<Modal>
<RegularCycles/>
<AdHocAndReleased/>
<Container>
<Button>Cancel</Button>
<Button>Create draft pay run</Button>
</Container>
</Modal>
```

Reply:
Having this component makes the code way more complicated, strip the logic into the other two files.